### DISCUSSION TITLE
Installation Issue when installation via the .sh scripts (md5 mismatch)

### DISCUSSION THREAD

**Discussion by arsenomadridabin** [deepmodeling/deepmd-kit#3710] [Q&A]
Tried installing GPU version of latest binaries with this script "(https://github.com/deepmodeling/deepmd-kit/releases/download/v2.2.10/deepmd-kit-2.2.10-cuda118-Linux-x86_64.sh.0)".  Apparently, the script requires script name to end in .sh. Therefore, I had to rename the script from xxx.sh.0 to xxx.sh . In doing so, because of the change in file name, I get an md5 mismatch issue. I think this a deadlock situation or am I missing somehintg? Could you please guide me on this.

**Comment by njzjz** [comment]
Please see #740 or documentation

**Reply by arsenomadridabin** [reply]
Thanks, this helped.
